packages/website/components/account/filesManager/fileRowItem.js

```diff
@@ -125,9 +125,14 @@ const FileRowItem = props => {
           <Tooltip content={fileRowLabels.cid.tooltip} />
           {fileRowLabels.cid.label}
         </span>
-        <span className="cid-truncate medium-up-only">
+        <a
+          className="cid-truncate underline medium-up-only"
+          href={`//ipfs.io/ipfs/${cid}`}
```

Comment:
dweb.link is the preferred gateway domain these days, as it enforces origin isolation per cid by redirecting to a cid subdomain. It also deals with cidv0 to cidv1 rewriting as required.
```suggestion
          href={`https://dweb.link/ipfs/${cid}`}
```